lib/helpers.rb

```diff
@@ -135,13 +148,17 @@ def reduce_aggs meta, options={}
       relation_types = meta.fetch("relationTypes",[])
```

Comment:
@kjgarza have you looked into only fetching the relation-types we want from the API, instead of filtering them out later?